src/functional/iterator/transforms/inline_lifts.py

```diff
@@ -3,35 +3,77 @@
 
 
 class InlineLifts(NodeTranslator):
+    @staticmethod
+    def _is_lift(node: ir.Node):
+        return (
+            isinstance(node, ir.FunCall)
+            and isinstance(node.fun, ir.FunCall)
+            and isinstance(node.fun.fun, ir.SymRef)
+            and node.fun.fun.id == "lift"
```

Comment:
Eve-nodes became comparable at some point (the ID-field was dropped), so `isinstance(node.fun.fun, ir.SymRef) and node.fun.fun.id == "lift"` can be shortened to `node.fun.fun == ir.SymRef(id="lift")`.